### DISCUSSION TITLE
Data Kiosk Economics create query "unauthorized access"

### DISCUSSION THREAD

**Discussion by kevnhowe** [amzn/selling-partner-api-models#4016] [General] [answered]
Getting Unauthorized Access error when trying to call CreateQuery method.
      "code": "Unauthorized",
      "message": "Unauthorized to access requested fields.",
      "details": "Unauthorized to access: [analytics_economics_2024_03_15.economics.cost.miscellaneousCost.amount, analytics_economics_2024_03_15.economics.cost.fbaCost.shippingToAmazonCost.amount, analytics_economics_2024_03_15.economics.cost.costOfGoodsSold.amount]"

Is there a developer permissions section for DataKiosk that I must allow in order to call these fields?

**Comment by mafge** [comment, marked as answer]
Hi @kevnhowe,
thanks for reaching out! There was a known issue with the economics schema where it was not possible to create a query despite having all required roles. This issue is resolved by now - can you try again on your side?
Best,
Marc
Selling Partner Developer Services

**Reply by kevnhowe** [reply]
This is fixed.

**Comment by ashish-atidiv** [comment]
I encountered the same issue again. However, it got resolved with a new refresh token.
